launch.sh

```diff
@@ -155,17 +156,36 @@ launch_system_tests() {
 # Deploy a Virtual machine on the cluster and launch cukinia tests in it.
 # Fetch results
 launch_vm_tests() {
+  cd ansible
+
   # Add VM inventory file
   # This file cannot be added at the beginnig of launch.sh cause it is used
   # only during thes step
-  ANSIBLE_INVENTORY="${ANSIBLE_INVENTORY},${INVENTORY_VM}"
-  CQFD_EXTRA_RUN_ARGS="${CQFD_EXTRA_RUN_ARGS} -e ANSIBLE_INVENTORY=${ANSIBLE_INVENTORY} -v /etc/seapath-ci/vm_file:${WORK_DIR}/ansible/vm_images"
-
-  cd ansible
+  ANSIBLE_INVENTORY_VM="${ANSIBLE_INVENTORY},${INVENTORY_VM_CLUSTER}"
+  CQFD_EXTRA_RUN_ARGS="${CQFD_EXTRA_RUN_ARGS} -e ANSIBLE_INVENTORY=${ANSIBLE_INVENTORY_VM} -v /etc/seapath-ci/vm_file:${WORK_DIR}/ansible/vm_images"
   cqfd run ansible-playbook \
   --key-file "${PRIVATE_KEYFILE_PATH}" \
   playbooks/deploy_vms_cluster.yaml
-  echo "test VM deployed successfully"
+  echo "test VM deployed successfully on cluster"
+
+  cqfd run ansible-playbook \
```

Comment:
You have also added a VMs configuration part?
If it is the case, mention it in the commit message.

Reply:
ok